app/views/flags/history.html.erb

```diff
@@ -1,7 +1,7 @@
 <h1>Flag History</h1>
 <p>
   Below is a list of all the flags you have cast and the outcome of each of them. Moderators handling your flags may also
-  have left you feedback.
+  have left you a feedback.
```

Comment:
"Feedback" is a mass noun not a counting noun.  "A feedback" sounds odd to me.  "Feedback" or "a response" both work.

Reply:
OK. rolled back to "Feedback".